clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -114,6 +117,69 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(config_path):
+            """ if a configuration file exists,
+            parse it and accordingly deploy models and applications
+
+            Parameters
+            ----------
+            config_path : str
+                Path to the configuration file.
+            """
+            if (config_path):
+                with open(config_path, "r") as stream:
+                    config = yaml.load(stream)
+                    if ("links" in config):
+                        for link in config["links"]:
+                            try:
+                                self.link_model_to_app(
+                                    config["links"][link]["app_name"],
+                                    config["links"][link]["model_name"])
+                            except ClipperException as e:
+                                print("required parameters not found")
+                                raise e
+                    if ("applications" in config):
+                        for application in config["applications"]:
+                            try:
+                                input_type = config["applications"][
+                                    application]["input_type"]
+                                slo = config["applications"][application][
+                                    "slo"]
+                                default_value = config["applications"][
+                                    application]["default_value"]
+                                self.register_application(
+                                    application, input_type, default_value,
+                                    slo)
+                            except ClipperException as e:
```

Comment:
Catch `KeyErrors` here too and throw an appropriate `ClipperException`

Reply:
Done